packages/create-sitecore-jss/src/templates/nextjs-multisite/scripts/config/plugins/multisite.ts

```diff
@@ -13,12 +13,14 @@ class MultisitePlugin implements ConfigPlugin {
 
   async exec(config: JssConfig) {
     let sites: SiteInfo[] = [];
-    const endpoint = process.env.GRAPH_QL_ENDPOINT || config.graphQLEndpoint;
-    const apiKey = process.env.SITECORE_API_KEY || config.sitecoreApiKey;
+
+    // graphQL endpoint can have a dynamic value in the config - so we resolve it in a special way at build time
+    const endpoint = config.graphQLEndpoint;
+    const apiKey = config.sitecoreApiKey;
 
     if (!endpoint || !apiKey) {
       console.warn(
-        chalk.yellow('Skipping site information fetch (missing GraphQL connection details)')
+        chalk.yellow('Skipping site information fetch (missing GraphQL connection details). Endpoint or API key missing.')
```

Comment:
Can simplify this I think
```suggestion
        chalk.yellow('Skipping site information fetch (missing GraphQL endpoint or API key)')
```